jwt/decode-jwt.js

```diff
@@ -0,0 +1,27 @@
+"use strict";
+
+require('dotenv').config({path: __dirname + '/../.env'});
+
+const express = require('express')
+const jwt = require('jsonwebtoken')
+
+const app = express();
+
+const app_public_key = `-----BEGIN PUBLIC KEY-----
```

Comment:
In the case of our signed callbacks, we are using the Signature Key from their dashboard, not the public key. 
```suggestion
const signature_secret = VONAGE_SIGNATURE_SECRET;
```